test-data/projects/e2e-sub-projects/.joyride/src/test_runner.cljs

```diff
@@ -0,0 +1 @@
+../../../../../src/extension-test/e2e-test/.joyride/src/test_runner.cljs
```

Comment:
How are these cljs files with only file paths used?

Reply:
They are symlinks. Joyride has a fixed class path, so the code in these needs to be duplicated.